kafka-plugins-client/src/main/java/io/cdap/plugin/source/KafkaStreamingSource.java

```diff
@@ -16,6 +16,7 @@
 
 package io.cdap.plugin.source;
```

Comment:
this is an existing issue, but the package should be something like io.cdap.plugin.kafka.source. Otherwise we end up exporting some very broad io.cdap.plugin package which can conflict with other plugins. We can fix this in a separate PR.

Reply:
Will create a follow up PR for this.